tests/group/group_common.h

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+//
+//  SYCL 1.2.1 Conformance Test Suite
+//
+//  Copyright:	(c) 2018 by Codeplay Software LTD. All Rights Reserved.
+//
+// Provides common methods for group tests
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_GROUP_GROUP_COMMON_H
+#define __SYCLCTS_TESTS_GROUP_GROUP_COMMON_H
+
+#include "../common/common.h"
+#include <array>
+
+namespace {
+
+/** Retrieve group objects and store them
+ */
+template <typename kernelT, int numDims, size_t numItems>
+void store_group_instances(
+    std::array<cl::sycl::group<numDims>, numItems>& items)
+{
+  using item_t = cl::sycl::group<numDims>;
+
+  const auto oneElemRange =
+      sycl_cts::util::get_cts_object::range<numDims>::get(1, 1, 1);
+  const auto itemRange =
+      sycl_cts::util::get_cts_object::range<numDims>::get(numItems, 1, 1);
+
+  cl::sycl::buffer<item_t> itemBuf(items.data(),
```

Comment:
I was somehow puzzled by this.
Actually it works since the data in a buffer is not initialized and so there is no need to have a default constructor.